mne/io/egi/egimff.py

```diff
@@ -213,20 +212,31 @@ def _read_header(input_fname):
         version = np.fromfile(fid, np.int32, 1)[0]
     # Proposed change
     # the  line of code just below parses the string  in
-    # mff_hdr['date'] as a datetime object localised in UTC
+    # mff_hdr['date'] as a datetime object
+    #the datetime object gets localized in UTC at line ~458
     # You can delete this comment after reviewing.
+    
+    '''
+    the datetime.strptime .f directive (milleseconds) 
+    will only accept up to 6 digits. if there are more than
+    six millesecond digits in the provided timestamp string
+    (i.e. because of trailing zeros, as in test_egi_pns.mff)
+    then slice both the first 26 elements and the last 6
+    elements of the timestamp string to to truncate the 
```

Comment:
```suggestion
    elements of the timestamp string to truncate the 
```

Reply:
Hi @agramfort sorry, I just saw this, I fixed the style errors in my code and pushed  them to my forked branch _before_ I read your comment, sorry if that will cause any trouble. Sorry to say that i'm very new to github, I might have a hard time keeping up.

Where can I find the latest.inc file? Thanks for your help!

Reply:
doc/changes/latest.inc

Reply:
@larsoner @agramfort done, though it seems my changes to latest.inc are failing some of your tests?.

Sorry for the delay on getting back to this. And clarify, the long comments in the code ~220 can be removed after review if you wish. If there's anything else I missed and need to address just let me know. 

Cheers,

Scott.